src/components/manifold-subscription-details/components/View/index.tsx

```diff
@@ -49,11 +51,21 @@ export const View = () => {
   const { view } = store.state;
 
   if (view.isLoading) {
-    return <div>Loading...</div>;
+    return <SkeletonDetails />;
+  }
+
+  if (view.errors.length > 0) {
+    return [
+      ...view.errors.map(error => <Message type="error">{error.message}</Message>),
+      <SkeletonDetails />,
+    ];
```

Comment:
Something I’ve wondered about… is it better to display the skeleton screen with errors surrounding it, or on top of it? I don’t think it’s a huge detail, and obviously if you’re seeing these errors that’s the bigger issue. I just think persisting the component skeleton no matter what makes it “feel” more stable to me but can’t put my finger on it.

But at any rate, that’s a tiny design opinion, not PR feedback. I think this PR is definitely an improvement.